pulsar-broker/src/test/java/org/apache/pulsar/functions/worker/PulsarFunctionTlsTest.java

```diff
@@ -233,6 +240,11 @@ public void testFunctionsCreation() throws Exception {
                 functionConfig, jarFilePathUrl
             );
 
+            // Function creation is not strongly consistent, so this test can fail with a get that is too eager and
+            // does not have retries.
+            final PulsarAdmin admin = pulsarAdmins[i];
+            Awaitility.await().untilAsserted(() -> admin.functions().getFunction(testTenant, "my-ns", functionName));
```

Comment:
```suggestion
            Awaitility.await().ignoreExceptions().untilAsserted(() -> admin.functions().getFunction(testTenant, "my-ns", functionName));
```
by default, Awaitility won't retry if the code throws an exception other than AssertionError. You need to specify `.ignoreExceptions()` to achieve the desired behavior.
docs for `ignoreExceptions()`: https://github.com/awaitility/awaitility/wiki/Usage#ignoring-exceptions

Reply:
I didn't realize it worked that way. The test started passing after I added this block, so I incorrectly assumed it was right. Thanks!